products/BellHybrid/apps/application-bell-whats-new/ApplicationWhatsNew.cpp

```diff
@@ -31,6 +31,14 @@ namespace app
         cpuSentinel->BlockWfiMode(false);
     }
 
+    sys::MessagePointer ApplicationWhatsNew::handleAppClose([[maybe_unused]] sys::Message *msgl)
```

Comment:
```suggestion
    sys::MessagePointer ApplicationWhatsNew::handleAppClose(sys::Message *msgl)
```
`msgl` is passed to base class method